index.js

```diff
@@ -37,7 +42,24 @@ function getCartTotal(cart) {
 */
 function filterProductsByPriceRange(products, min, max) {
   const result = [];
+
+  if (!products.length) {
+    throw "No products scanned";
+  }
+  if (typeof min !== "number" || typeof max !== "number") {
+    throw "Price invalid";
+  }
+  if (min > max) {
+    throw "Price invalid";
```

Comment:
```suggestion
    throw "Price invalid. Min price cannot be greater than max price";
```
It's great that your error messages are appropriate for the error that they're referring to! You could make all of these "Price invalid" messages even **_more_** descriptive. Clear and helpful messages are really important for overall user experience (UX). That's really above and beyond, but always a good thing to think about. Great job on this lab :)